crates/bifrost/src/providers/replicated_loglet/remote_sequencer.rs

```diff
@@ -228,26 +233,32 @@ where
 #[derive(Clone)]
 struct RemoteSequencerConnection {
     inner: WeakConnection,
-    tx: mpsc::UnboundedSender<RemoteInflightAppend>,
+    inflight: mpsc::Sender<RemoteInflightAppend>,
```

Comment:
I'm not entirely convinced that this in general is a good design. A different approach can be to have one task associated with the remote sequencer to handle all responses and we use refactor RpcRouter to resolve the receive token when the connection is dropped.